packages/core/src/lifecycle.ts

```diff
@@ -57,3 +65,17 @@ export function asLifeCycleObserver<T = unknown>(binding: Binding<T>) {
 export function lifeCycleObserverFilter(binding: Binding<unknown>): boolean {
   return binding.tagMap[CoreTags.LIFE_CYCLE_OBSERVER] != null;
 }
+
+/**
+ * Sugar decorator to mark a class as life cycle observer
+ * @param group Optional observer group name
+ * @param scope Optional binding scope
+ */
+export function lifeCycleObserver(group = '', scope?: BindingScope) {
```

Comment:
I am concerned about flexibility of this API. Have you considered to accept the same arguments as `@bind` does?

```ts
export function lifeCycleObserver(group = '', ...specs: BindingSpec[]) {
  return bind(
    asLifeCycleObserver,
    {tags: [CoreTags.LIFE_CYCLE_OBSERVER_GROUP]: group},
    ...specs
  );
}
```

Reply:
👍